content/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lifecycle badges
+---
+
+![](/images/lifecycle/lifecycle.png)
+
+### Experimental ![](https://img.shields.io/badge/lifecycle-experimental-orange.svg) {#experimental}
+
+Very early days, a lot of churn in search of a good API.  Not on CRAN. Might not go anywhere. Use with care.
```

Comment:
An experimental package is in the very early stages of development. The API will be changing frequently as we rapidly iterate and explore variations in search of the best fit. Experimental packages will make API breaking changes without deprecation, so you are generally best off waiting until the package is more mature before you use it. Experimental packages will not be released on CRAN

Reply:
Do we need to explain what API is?

Reply:
Great. Explain as in spell it out because it's first acronym use, or as in _explain_ explain?

Reply:
EXPLAIN EXPLAIN

Reply:
That seems like it would be kind of a lot of not-so-lifecycle-related content at the start…but, I'm also not a succinct explainer of things. Is there a good tidyverse definition of API somewhere?

Reply:
Maybe an intro paragraph like:

This page describes the typical lifecycle on an R package. Knowing where a package is in its lifecycle is particularly important for understand how the API will change over time. The API (short for application programming interface) is the set of functions (and their arguments) that defines how you interact with the package. 

There are three ways an API can change:

1. It can __grow__ when a function or argument is added.
2. It can __shrink__ when a function or argument is removed.
3. It can __change__ when the meaning of an argument changes, or the type of data returned from a function changes.

Changing or shrinking the API will __break__ it: code that previously worked will no any longer. (Sometimes code will break even if the API doesn't; for example you might have accidentally depended on behaviour that the author thought was a bug)

Reply:
Added.